src/tooltip/tooltip.js

```diff
@@ -246,10 +246,6 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
                 }
               });
 
-              tooltipLinkedScope.$watch(function () {
-                $timeout(positionTooltip, 0, false);
-              });
-
               if (options.useContentExp) {
                 tooltipLinkedScope.$watch('contentExp()', function (val) {
```

Comment:
You might also need to schedule a reposition over here since if `contentExp()` changes, the content changes as well.